CONTRIBUTING.md

```diff
@@ -25,13 +25,22 @@ aiconfig/
 ├─ editor/
 ├─ aiconfig-docs/
 ├─ python/
-│   ├── model-parsers/
 ├─ typescript/
-│   ├── model-parsers/
 ├─ extensions/
 ```
 
-> *To Do: Add information on library structure and contribution opportunities*
+1. **{ Model Parsers }**: Model Parsers manage the conection to foundation models. By default, we provide the OpenAI models and Google PaLM models. The model parsers are defined in both `python` and `typescript`. 
+   
+    Instructions for contributing a Model Parser:
+    1. Add a folder under `aiconfig/extensions`
```

Comment:
Just extensions, not aiconfig/

Reply:
Oh, interesting -- was thinking we'd have an aiconfig/extensions repo similar to other libraries' "hub" repos. I think there's lots of effort required to set something like that up, though, so this should work for now